src/commcare_cloud/ansible/roles/kinesis_agent/templates/agent.json.j2

```diff
@@ -0,0 +1,14 @@
+{
+  "cloudwatch.emitMetrics": true,
+  "kinesis.endpoint": "{{ kinesis_endpoint }}",
+  "firehose.endpoint": "{{ firehose_endpoint }}",
+
+  "flows": [
+    {% for flow in kinesis_flows -%}
+    {
+      "filePattern": "{{ flow.file_pattern }}",
+      "deliveryStream": "{{ flow.delivery_stream }}"
+    }
```

Comment:
nit: if there were more than one then you'd need to insert a comma here on all but the last one